content/enterprise_influxdb/v1.10/concepts/schema_and_data_layout.md

```diff
@@ -276,4 +276,5 @@ Other factors to consider before setting shard group duration:
 Bulk insertion of historical data covering a large time range in the past will trigger the creation of a large number of shards at once.
 The concurrent access and overhead of writing to hundreds or thousands of shards can quickly lead to slow performance and memory exhaustion.
 
-When writing historical data, we highly recommend temporarily setting a longer shard group duration so fewer shards are created. Typically, a shard group duration of 52 weeks works well for backfilling.
+When writing historical data, we highly recommend temporarily setting a longer shard group duration so that fewer shards are created. Typically, a shard group duration of 30 days works well for backfilling, however you will need to consider whether the shards are likely to be affected by backfills again in future: if so, a slightly shorter duration may be preferable to reduce the resource cost of subsequent recompaction.
```

Comment:
```suggestion
When writing historical data, set a longer shard group duration so that InfluxDB creates few shards. Typically, a shard group duration of 30 days works well for backfilling, however consider whether the shards are likely to be affected by backfills again in the future. If so, a slightly shorter duration may be preferable to reduce the resource cost of shard recompaction.
```